js/src/modal.js

```diff
@@ -21,17 +21,23 @@ const EVENT_KEY          = `.${DATA_KEY}`
 const DATA_API_KEY       = '.data-api'
 const JQUERY_NO_CONFLICT = $.fn[NAME]
 const ESCAPE_KEYCODE     = 27 // KeyboardEvent.which value for Escape (Esc) key
+const LEFT_KEYCODE       = 37
+const RIGHT_KEYCODE      = 39
 
 const Default = {
+  autofocus: 'notTouch',      // true|false|notTouch
   backdrop : true,
   keyboard : true,
+  keyboardBtnNav: true,  // ability to use arrows to nav button focus
```

Comment:
renamed to `keyboardNav`..   wanting to keep option to use "arrow-key button nav" separate from <kbd>esc</kbd> key..   perhaps boostrap 5.0 can use less generic `keyboardEsc` (instead of `keyboard`) along with `keyboardNav`